### PR TITLE
chore: upgrade package dependencies

### PR DESCRIPTION
**Pull request by barlock**
Upgrading (most) of the packages dependencies. 

Particularly serialport has some new changes that I need and conflicts between new and old versions were causing issues in my packaging.

I purposefully left out a socket.io b/c of another bug I'm hunting down.

### CODE REVIEW

File: src/DMX.ts [hunk 1]
```diff
@@ -8,56 +8,56 @@ export interface DmxArgs {
 }
 
 export class DMX extends EventEmitter {
-    private readonly _devices: Devices;
-    private readonly _universesByName: Map<string, IUniverseDriver> = new Map();
-    constructor(options?: DmxArgs) {
-      super();
-      const devices = options?.devices ?? {};
+  private readonly _devices: Devices;
```
Comment (barlock) [inline src/DMX.ts:11]: These are all whitespace changes that happened when I ran `eslint --fix`

**Comment by hrueger** [comment]
Hi @barlock ,
Thanks a lot for the contribution 🎉

Lgtm, I'll test it in the next couple of days when I have time and then merge it 👍

**Comment by hrueger** [comment]
Hi @barlock,
Sorry for the late response - somehow I completely forgot about this...

**Comment by hrueger** [comment]
I enabled the socket.io compatibility mode, so it should still work with older clients 👍

**Comment by hrueger** [comment]
This is now released as 0.3.0 🎉

**Pull request merged**